File: lua/calc.lua
io.write("Enter the first number: ")
a = io.read()
io.write("Enter the second number: ")
b = io.read()

a = tonumber(a)
b = tonumber(b)

print(string.format("%g+%g=%g", a, b, a + b))
print(string.format("%g-%g=%g", a, b, a - b))
print(string.format("%g*%g=%g", a, b, a * b))
print(string.format("%g/%g=%g", a, b, a / b))

File: java/Calc.java
import java.util.Scanner;

public class Calc {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);

    float a, b;

    System.out.print("Enter the first number: ");
    a = input.nextFloat();
    System.out.print("Enter the second number: ");
    b = input.nextFloat();

    System.out.printf("%g+%g=%g\n", a, b, a + b);
    System.out.printf("%g-%g=%g\n", a, b, a - b);
    System.out.printf("%g*%g=%g\n", a, b, a * b);
    System.out.printf("%g/%g=%g\n", a, b, a / b);

    input.close();
  }
}
